nym-vpn-desktop/src-tauri/src/commands/node_location.rs

```diff
@@ -89,164 +89,74 @@ pub async fn get_node_location(
 pub async fn get_countries(node_type: NodeType) -> Result<Vec<Country>, CmdError> {
     debug!("get_countries");
     match node_type {
-        NodeType::Entry => get_entry_countries().await,
-        NodeType::Exit => get_exit_countries().await,
+        NodeType::Entry => get_gateway_countries(false).await,
+        NodeType::Exit => get_gateway_countries(true).await,
     }
 }
```

Comment:
I would suggest we try to fetch both of these in the same call to avoid reaching out to harbourmaster twice (it's unfortunately slow). I think you should be able to use `GatewayClient::lookup_described_entry_and_exit_gateways_with_location()`.

Reply:
If this works, I'll change for uniffi as well.

Reply:
I think this is an optimization we will need to make later. It requires too much change on the clients to make this change right now.